### DISCUSSION TITLE
v2.2.1

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3395] [分享 / Share]
- 修复部分浏览器 / 脚本管理器中脚本无法运行. (对应报错 `Error: Couldn't find a style target`)
- 移除代码中的零宽空格. (#3391)

报错是 `_ is not defined` 的请看 #3384 关于 jsDelivr 被墙的问题.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.2.1'>v2.2.1</a>.</em>

**Comment by xiaohuangsu** [comment]
直接没了，脚本加载了，页面还是B站原版的，左边也没有设置

**Reply by Geeyun-JY3** [reply]
请
善
用
搜
索
，
看
置
顶
#3331

**Comment by Geeyun-JY3** [comment]
话说「ⓘ关于」里的版本号是从 v2.2.1 开始不带最后一个 commit SHA hash 吗？

**Reply by the1812** [reply]
和 `v2.2.1` 所指向的 commit 相同时会省略
